alembic/testing/requirements.py

```diff
@@ -140,3 +140,14 @@ def comments_api(self):
     @property
     def alter_column(self):
         return exclusions.open()
+
+    @property
+    def db_computed_columns(self):
+        # TODO: should import the requirements from sqlalchemy
+        return exclusions.skip_if(["postgresql < 12", "sqlite", "mysql < 5.7"])
```

Comment:
@zzzeek here I should ideally return the value of `sqlalchemy.testing.requirements.SuiteRequirements.computed_columns` but currently it defines a `closed` condition. 
The real condition is set in `test.requirements.DefaultRequirements.computed_columns` which I cannot import here

Maybe the condition to indicate if a backend supports computed columns should be somewhere in the `sqlalchemy.testing` library so that it can be imported?